v8js_v8object_class.cc

```diff
@@ -210,7 +210,13 @@ static HashTable *v8js_v8object_get_properties(zval *object) /* {{{ */
 	v8js_v8object *obj = Z_V8JS_V8OBJECT_OBJ_P(object);
 
 	if (obj->properties == NULL) {
+#if PHP_VERSION_ID >= 70300
+		zend_gc_status gc_status;
+		zend_gc_get_status(&gc_status);
+		if (gc_status.runs) {
```

Comment:
`runs` is the number of GC runs, not whether it's currently running. The right way to handle this is to implement a `get_gc()` handler, which is called instead of `get_properties()` during GC.